spock-spring/src/main/java/org/spockframework/spring/SpringMockTestExecutionListener.java

```diff
@@ -36,12 +37,19 @@
  *
  * @author Leonard Bruenings
  */
-public class SpringMockTestExecutionListener extends AbstractSpringTestExecutionListener {
+public class SpringMockTestExecutionListener extends AbstractSpringTestExecutionListener implements Ordered {
 
   public static final String MOCKED_BEANS_LIST = "org.spockframework.spring.SpringMockTestExecutionListener.MOCKED_BEANS_LIST";
 
   private final MockUtil mockUtil = new MockUtil();
 
+  @Override
+  public int getOrder() {
+    // needs to run before the DependencyInjectionTestExecutionListener
+    // https://github.com/spockframework/spock/issues/1790
+    return 1955;
```

Comment:
This is very arbitary, but I guess that is for the other order `2000` in class `DependencyInjectionTestExecutionListener`.